src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CreateCollectionCommand.java

```diff
@@ -59,6 +61,19 @@ public record Options(
               implementation = IndexingConfig.class)
           IndexingConfig indexing) {
 
+    public record IdConfig(
+        @Nullable
+            @Pattern(
+                regexp = "(objectId|uuid|uuidv6|uuidv7)",
+                message = "Id type can only be 'objectId', 'uuid' , 'uuidv6' or 'uuidv7'")
```

Comment:
Just for completeness, what uuid version is does `uuid` correspond to? Seems a little strange that we have 'uuid' and then 'uuidv6' + 'uuidv7'

Reply:
that'd be v4 (random based). I think naming is due to it being the default choice, in absence of other indicators.
/cc @amorton